### DISCUSSION TITLE
Various Email Error Messages

### DISCUSSION THREAD

**Discussion by nickabond** [PHPMailer/PHPMailer#2672] [Q&A] [answered]
Sometimes when we send an email a user may not exist because someone typed in the email wrong at our company. There are other errors that we could get as well, but that's a common example. Does PHPMailer provide a way to get this error?

Thanks!

Nick

**Comment by long76** [comment, marked as answer]
> 



> Sometimes when we send an email a user may not exist because someone typed in the email wrong at our company. There are other errors that we could get as well, but that's a common example. Does PHPMailer provide a way to get this error?
> 
> Thanks!
> 
> Nick

Hi, PHPMailer don't have this functional. Usual only Mail server can check mail on exist and send error message to sender. Idk IETF protocol to mail server which include service packet on check exit mail. This may call DDoS attack.

**Reply by nickabond** [reply]
Okay, gotcha, so basically PHP Mailer is just checking did the request go out to the sending mail server successfully and it doesn't actually know whether it made it to the actual recipient or not?

**Reply by long76** [reply]
> Okay, gotcha, so basically PHP Mailer is just checking did the request go out to the sending mail server successfully and it doesn't actually know whether it made it to the actual recipient or not?

Correct

**Reply by nickabond** [reply]
Thanks very much!

**Reply by Synchro** [reply]
This is inherent in the way that SMTP works. The server that you submit your message to (successfully) is quite likely to have no idea whether the intended recipient exists or not. Eventually you will find out by receiving a bounce message, but that may not happen for days.
